torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -144,6 +195,8 @@ def parallelize_llama(model, world_mesh, parallel_dims, job_config: JobConfig):
             raise NotImplementedError(
                 "fused_rmsnorm not yet compatible with TP. Please use layernorm or rmsnorm."
             )
+        if parallel_dims.dp_replicate_enabled:
+            raise NotImplementedError("DDP/HSDP + TP are not supported yet.")
```

Comment:
We should make DDP + TP work and see if it could support llama3_8b or llama2_7b. If not, we could try to import other models instead of Llama, and have DDP to apply to that model instead :)